packages/orbit-components/src/primitives/ButtonPrimitive/index.d.ts

```diff
@@ -5,6 +5,8 @@ import * as Common from "../../common/common";
 type inexactString = string | null | undefined;
 type functionReturningString = () => string;
 
+export type Size = "small" | "normal" | "large";
```

Comment:
Why is this being exported?

Reply:
It's consumed in e.g. `ButtonLink/index.d.ts`